po/zh_CN.po

```diff
@@ -100,15 +100,15 @@ msgstr "Copyright © 2017 Andy Holmes"
 
 #: src/prefs.js:81
 msgid "This program comes with absolutely no warranty."
-msgstr "这个程序没有绝对保证."
+msgstr "中文翻译者: <a href=\"https://github.com/plexpt\">Plexpt</a>"
```

Comment:
It’s not the place to credit translators. @andyholmes, how about adding proper [translator-credits](https://developer.gnome.org/gtk3/stable/GtkAboutDialog.html#gtk-about-dialog-set-translator-credits)?